books.js

```diff
@@ -0,0 +1,111 @@
+const bookList = [];
+
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const section = document.querySelector('.bookings');
+let count = 0;
+
+function addBook(Title, Author) {
+  const booky = new Books(Title, Author);
+  bookList.push(booky);
+
+  const container = document.createElement('div');
+  const names = document.createElement('h3');
+  const authors = document.createElement('h4');
+  const button = document.createElement('button');
+  const line = document.createElement('span');
+
+  section.append(container);
+  container.append(names);
+  container.append(authors);
+  container.append(button);
+  container.append(line);
+
+  const counting = `count${count}`;
+  const buttoning = `button${count}`;
+
+  container.setAttribute('class', 'booksection');
+  container.setAttribute('id', counting);
+  names.innerText = booky.title;
+  authors.innerText = booky.author;
+  button.setAttribute('class', 'removebutton');
+  button.setAttribute('id', buttoning);
+  button.innerHTML = 'Remove';
+  line.setAttribute('class', 'line');
+  count += 1;
+}
+
+function removeList(name, author) {
+  for (let x = 0; x < bookList.length; x += 1) {
+    if ((bookList[x].title === name) && (bookList[x].author === author)) {
+      bookList.splice(x, 1);
+    }
+  }
+}
+
+function removeBook(args) {
+  const removing = document.querySelectorAll('.removebutton');
+  let count1 = 0;
+  removing.forEach(() => {
+    if (args === (`button${count1}`)) {
+      const ids = `count${count1}`;
+      const nam = `#count${count1} h3`;
+      const auth = `#count${count1} h4`;
+      const nodes = document.getElementById(ids);
+      const names = document.querySelector(nam);
+      const author = document.querySelector(auth);
+      removeList(names.innerHTML, author.innerHTML);
+      nodes.parentNode.removeChild(nodes);
+    }
+    count1 += 1;
+  });
+}
```

Comment:
- [ ] good job setting up the project 👍 however after populating and refreshing the list the user is not able to remove the existing books anymore. when adding a `console.log()` in the `addEventListener` for removing the item it looks like the event is not called on click.